socket.h

```diff
@@ -19,12 +19,14 @@ class cSatipSocket {
   bool useSsmM;
   in_addr_t streamAddrM;
   in_addr_t sourceAddrM;
+  size_t rcvBufSizeM;
   bool CheckAddress(const char *addrP, in_addr_t *inAddrP);
   bool Join(void);
   bool Leave(void);
 
 public:
   cSatipSocket();
+  cSatipSocket(size_t rcvBufSizeP);
```

Comment:
Let's use constructor delegation in order not to duplicate the code:
```
cSatipSocket() : cSatipSocket(0) {}
cSatipSocket(size_t rcvBufSizeP);
```

Reply:
Done.